cookbooks/awscli/metadata.rb

```diff
@@ -0,0 +1,8 @@
+name              "awscli"
+maintainer        "OpenStreetMap Administrators"
+maintainer_email  "admins@openstreetmap.org"
+license           "Apache-2.0"
+description       "Installs and configures awscli"
+
+version           "1.0.0"
+supports          "ubuntu"
```

Comment:
Not sure how it work on debian with that? I thought chef would reject it...

Reply:
The kitchen tests did pass. Weird. Fixed. now.